spring-rabbit/src/main/java/org/springframework/amqp/rabbit/connection/AbstractConnectionFactory.java

```diff
@@ -556,6 +558,16 @@ public boolean hasPublisherConnectionFactory() {
 		return this.publisherConnectionFactory != null;
 	}
 
+	/**
+	 * Support backoff policy when get an empty channel from connection.
+	 * @param backOff {@link BackOff}
+	 * @since 3.1.3
+	 * @see BackOff#start()
```

Comment:
I don't see a reason in this link.
Why is that `{@link BackOff}` in the param not enough?
Why to be so specific for its `start()` method?

Reply:
I actually want to instruct developers on how to use backoff, but this seems somewhat misaligned with the focus on enhancements. I've made some modifications. Could you please confirm them? Thank you.